hack/verify-gofmt.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+echo "Verifying gofmt"
+
+diff=$(find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -d 2>&1)
```

Comment:
This will verify everything (which is good) but I think there's currently some code under `/cmd/cyclonus` that needs to be fixed (it was failing running on my local

Reply:
I added this as a temp for now to get the tests in, we should remove it once the code under `/cmd/cyclonus` is fixed

https://github.com/kubernetes-sigs/network-policy-api/pull/47/commits/b0cafd0443c82e58d1d6bac4a17c305d1c12dd61

Reply:
Sweet! Can you make an issue to track that @olivercodes?

Reply:
https://github.com/kubernetes-sigs/network-policy-api/issues/48